npc/re/quests/eden/eden_common.txt

```diff
@@ -25,7 +25,7 @@
 
 // Common NPCs :: lv_tre
 //============================================================
-moc_para01,27,35,5	script	Secretary Lime Evenor	4_F_EDEN_OFFICER,{
+moc_para01,27,35,5	script	Secretary Laime Evenor	4_F_EDEN_OFFICER,{
```

Comment:
The illust name is Laime, and..
Lime is like mineral name, Laime is more common for human name.

Reply:
The illust name means nothing, a lot of file names are mispelled, 라임 is pronounced Lime (Lah-eem), not Laime (Lah-ee-mee)

Reply:
I would like to read second opinion,
thoughts @Lemongrass3110 / @aleos89 ?

Reply:
I think it's pretty clear
라임 means lime, https://ko.wikipedia.org/wiki/%EB%9D%BC%EC%9E%84
iRO translated it as Lime, and jRO as ライム that also means lime.

Reply:
No offense, but I would like dev's other opinion.

Reply:
All our sources suggest the NPC name is Lime.

Added other devs to the approval list for their input.

Reply:
From translations it pretty specifically states Lime. I wouldn't go with kRO's image-text translations because there are lots of old images with "Load Knight" and other misspellings. Some other staff looked at other official servers and their translations are also "Lime".

Reply:
Honestly I have no idea how the name should be translated. I don't mind one or the other since it doesn't change the script comprehension. Maybe we shouldn't change it when it's too ambiguous, but you're more qualified than me to decide of the translation.

Reply:
Thanks guys, really appreciate your opinions.